pkg/storage/replica.go

```diff
@@ -1877,7 +1878,8 @@ func (r *Replica) tryAddWriteCmd(
 			// AmbiguousResultError indicates to caller that the command may
 			// have executed.
 			if tryAbandon() {
-				log.Warningf(ctx, "context cancellation of command %s", ba)
+				log.Warningf(ctx, "context cancellation after %s of attempting command %s",
+					time.Since(startTime), ba)
```

Comment:
Makes sense, done. It's far from standard in the code base though -- it's only done that way in one place, while there are a bunch of places printing durations as strings.